packages/ui-new/src/components/wui-icon/index.ts

```diff
@@ -7,7 +7,6 @@ import { customElement } from '../../utils/WebComponentsUtil.js'
 import styles from './styles.js'
 
 // -- Svg's-------------------------------- //
-import { arrowTopRightSvg } from '../../assets/svg/arrow-top-right.js'
```

Comment:
quickly solved a few errors in this component since we exported the same icons twice